rust/src/measurements/report_noisy_max/test.rs

```diff
@@ -0,0 +1,79 @@
+use crate::error::Fallible;
+
+use super::*;
+
+#[test]
+fn test_rnm_gumbel() -> Fallible<()> {
+    let input_domain = VectorDomain::new(AtomDomain::default());
+    let input_metric = LInfDistance::default();
+    let de = make_report_noisy_max(
+        input_domain,
+        input_metric,
+        RangeDivergence,
+        1.,
+        Optimize::Max,
+    )?;
+    let release = de.invoke(&vec![1., 2., 3., 2., 1.])?;
+    println!("{:?}", release);
```

Comment:
as with the python tests, add little explanation for the future dev about what they should expect to see.